vhd/packer/configure-windows-vhd.ps1

```diff
@@ -49,8 +49,13 @@ function Get-ContainerImages {
                 "mcr.microsoft.com/windows/nanoserver:1809",
                 "mcr.microsoft.com/oss/kubernetes/pause:1.4.1",
                 "mcr.microsoft.com/oss/kubernetes/pause:3.4.1",
-                "mcr.microsoft.com/oss/kubernetes-csi/livenessprobe:v2.0.1-alpha.1-windows-1809-amd64",
-                "mcr.microsoft.com/oss/kubernetes-csi/csi-node-driver-registrar:v1.2.1-alpha.1-windows-1809-amd64")
+                "mcr.microsoft.com/oss/kubernetes/azure-cloud-controller-manager:v1.0.8",

```

Comment:
I think we don't need this, as the `cloud-controller-manager` pod will never be scheduled onto a Windows node.

Is there a Windows `cloud-node-manager` container image @marosset @jsturtevant ?

Reply:
Sorry I was copying the wrong image url, the one we need is cloud-node-manager, let me update

Reply:
yes for out of tree: https://github.com/kubernetes-sigs/cloud-provider-azure/blob/960a8ee5ae26dc1be4e2dbf53f4f865b28c72e6d/examples/out-of-tree/cloud-node-manager.yaml#L105